SETUP.md

```diff
@@ -67,9 +67,10 @@ python3 ups.py
 ***This section is NOT complete!!!***
 
 ```
-sudo cp startup/ups.service /lib/systemd/system/
-sudo chmod 644 /lib/systemd/system/ups.service
+sudo cp SuperUPS/host_monitor/ups*.py /usr/bin/
```

Comment:
Could we put these in /usr/local/bin instead of /usr/bin?

Reply:
Can do, will update here shortly